6/slim/Dockerfile

```diff
@@ -9,14 +9,21 @@ RUN apt-get update && \
     apt-get install -y -q --no-install-recommends \
         gcc \
         imagemagick webp coreutils gifsicle libvpx? \
-        libvpx-dev libimage-exiftool-perl libcairo2-dev \
-        ffmpeg libffi-dev \
+        libvpx? libvpx-dev libimage-exiftool-perl libcairo2 libcairo2-dev \
+        ffmpeg libffi6 libffi-dev \
         libcurl4-gnutls-dev libgnutls28-dev \
-        python-dev && \
+        python python-dev && \
+    pip install --no-cache-dir thumbor==6.6.0 envtpl==0.6.0 tc_aws && \
+    apt-get remove \
```

Comment:
It won't build, since the autoremove assumes an answer here: `apt-get remove -y \` should solve this.